submission/tree_genetic_programming/install.sh

```diff
@@ -0,0 +1,2 @@
+chmod 766 TreeGeneticProgramming
+apt-get install qtquickcontrols2-5-dev
```

Comment:
if you're just trying to satisfy a qt dependency, you might be able to find one from conda, https://anaconda.org/conda-forge/qt

Reply:
as mentioned in the guide your method must be installable without sudo permissions, so apt, yum etc aren't going to work